pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -26,6 +26,8 @@
   public static final String PREFIX_OF_SSL_SUBSET = "ssl";
   public static final String HTTP_PROTOCOL = "http";
   public static final String HTTPS_PROTOCOL = "https";
+  public static final String INSTANCE_FAILURE_DOMAIN = "pinot.environment.instance.failureDomain";
```

Comment:
IMO, Better place for this is in the PinotEnvironment interface. to have properties located with the interface itself.

Reply:
Updated.